resources/views/themes/default1/front/paynow.blade.php

```diff
@@ -121,68 +121,47 @@
 
 
                     </table>
-                    <hr class="tall">
-                    <!-- <h4 class="heading-primary">Cart Totals</h4> -->
-                   <!--  <div class="col-md-12">
-                        <table class="cart-totals">
-                            <tbody>
-
-
-                                <tr class="total">
-                                    <th>
-                                        <strong>Order Total</strong>
-                                    </th>
-                                    <td>
-                                        <strong><span class="amount"><small>{!! $symbol !!} </small> {{$invoice->grand_total}}</span></strong>
-                                    </td>
-                                </tr>
-
-                            </tbody>
-                        </table>
-                        <hr class="tall">
-                    </div> -->
 
                 </div>
-                {!! Form::open(['url'=>'checkout','method'=>'post']) !!}
+                {!! Form::open(['url'=>'checkout','method'=>'post','id' => 'checkoutsubmitform']) !!}
                   @if($invoice->grand_total > 0)
                 <h4 class="heading-primary">Payment</h4>
                     <?php $gateways = \App\Http\Controllers\Common\SettingsController::checkPaymentGateway($invoice->currency);
                       $rzpstatus = \App\Model\Common\StatusSetting::first()->value('rzp_status');
                        ?>
-                     @if($gateways) 
-                  <div class="form-group">
-
-                    <div class="col-md-6">
-                        @if(count($gateways)>0 ) 
-                  <div class="form-group">
+                    
+                
+                @if(count($gateways)>0 ) 
+                  <div class="row">
 
                     <div class="col-md-6">
                         @foreach($gateways as $gateway)
-                        {{ucfirst($gateway)}} {!! Form::radio('payment_gateway',strtolower($gateway)) !!}<br><br>
+                        <?php
+                          $processingFee = \DB::table(strtolower($gateway))->where('currencies',$invoice->currency)->value('processing_fee');
+                        ?>
+                        <input type="hidden" name="process_fee" value="{{$processingFee}}">
+                        <input type="radio"  data-currency="{{$processingFee}}" id="allow_gateway" name='payment_gateway'  value={{$gateway}}>
```

Comment:
please use `Form::radio` if functionality is simple enough

Reply:
Fixed